xmppserver/src/main/java/org/jivesoftware/openfire/http/HttpSessionManager.java

```diff
@@ -110,7 +111,7 @@ public void start() {
 
         // Periodically check for Sessions that need a cleanup.
         inactivityTask = new HttpSessionReaper();
-        TaskEngine.getInstance().schedule( inactivityTask, 30 * JiveConstants.SECOND, 30 * JiveConstants.SECOND );
+        TaskEngine.getInstance().schedule( inactivityTask, 30 * JiveConstants.SECOND, sessionCleanupCheck * JiveConstants.SECOND);
```

Comment:
Trivial, but I think the second param should be `sessionCleanupCheck * JiveConstants.SECOND` too.

Reply:
Certainly could be. I think that's the initial delay before beginning the periodic looping - we left it at 30 to "give time for things to settle". No probs if you'd prefer it made the same.